web/static/js/script.js

```diff
@@ -445,17 +447,15 @@ function initVolControl(ctrl) {
       } else {
         console.log('volume control ' + ctrl.id + ' not bound to any zone or group');
       }
-      req_throttled = true;
-      setTimeout(() => {
-        req_throttled = false
-      }, VOL_REQ_THROTTLE_MS);
-
+      last_req_stamp = cur_stamp; // only update on a successful request (avoids constant rejection of a stream of user requests)
+    } else {
+      console.log('volume adjustment rejected, last request made < 50ms ago')
```

Comment:
Was this intended to be left in? It tends to flood the console when dragging the volume sliders. Also, I'm no Javascript expert but if left in should it have a semicolon like the edit below adding one?